src/components/login.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { Login } from "../redux/actions/loginAction";
+import Button from "../views/Button/index";
+import LoginForm from "./LoginForm";
+
+class LoginPage extends React.Component {
+  constructor(props) {
```

Comment:
This constructor is irrelevant since the bind statement has been removed. Please refactor accordingly.